### DISCUSSION TITLE
How to ignore StyletIoCFindConstructorException

### DISCUSSION THREAD

**Discussion by wf-soft** [canton7/Stylet#301] [Q&A]
How to skip the unregistered styletiocfindconstructorexception and make it null when inject, so as to skip some unregistered types in the unit test

**Comment by canton7** [comment]
There is no support for this in StyletIoC

**Reply by wf-soft** [reply]
@canton7  OK, I've tried to create injection in unit test. I don't need to worry about this anymore. Thank you!
